flutter/realm_flutter/linux/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.19)
+cmake_minimum_required(VERSION 3.10)
```

Comment:
we require Cmake 3.21 now. https://github.com/realm/realm-dart#environment-setup-for-realm-flutter
I think we should keep the requirement.

Reply:
This is different - we're not building Core here, so there's no reason to use 3.21.

Reply:
Yeah, that's just the dummy flutter plugin - it doesn't build anything except the "hello world" plugin code. it doesn't need the newer CMake version.

Reply:
yes indeed